src/com/hms/flexyosisoftconnector/OSIsoftServer.java

```diff
@@ -398,6 +405,12 @@ public int initTags() throws JSONException {
     return retval;
   }
 
+  private void checkResponseCode(int res) {
+    if (res != NO_ERROR) {
+      Logger.LOG_WARN("The request was not succesful");

```

Comment:
```suggestion
      Logger.LOG_WARN("The request was not successful");
```

Reply:
What request? You may want to add more context to the debug messaging. this would be prints near the RequestHTTPS calls.

Reply:
Is the change I made what you had in mind?